vendor/redz/tlm-lw-profile-865.yaml

```diff
@@ -1,3 +1,9 @@
+# Vendor profile ID, can be freely issued by the vendor. NOTE: The vendor profile ID is different from the vendorID.
+# The vendor Profile ID should be an incremental counter for every unique device listed in the vendor's folder.
+# This vendor profile ID is also used on the QR code for LoRaWAN devices, see
+# https://lora-alliance.org/wp-content/uploads/2020/11/TR005_LoRaWAN_Device_Identification_QR_Codes.pdf
+vendorProfileID: 0
```

Comment:
Please remove all instances of `vendorProfileID`. This method has been replaced with profileIDs: https://github.com/TheThingsNetwork/lorawan-devices?tab=readme-ov-file#vendor-device-index